src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -207,8 +200,47 @@ public void run() {
     TypeModel type;
     if (baseIdentifier.equals(RESPONSE_PLACEHOLDER)) {
       view.variable(context.getNamer().getSampleResponseVarName(context));
+      boolean pageStreaming = context.getMethodConfig().getPageStreaming() != null;
+      boolean pageStreamingAndUseResourceName =
+          pageStreaming
+              && context
+                  .getFeatureConfig()
+                  .useResourceNameFormatOption(
+                      context.getMethodConfig().getPageStreaming().getResourcesFieldConfig());
```

Comment:
Do resource objects only apply to the top level of a response (paged response in this case), or could they also be defined for submessages within the response? In `library_gapic.yaml`, for example, the end section contains `- message_name` entries. Can those only refer to top level responses (either pages or unpaged), or could they apply to submessages within the responses?

@pongad @garrettjonesgoogle

Reply:
I don't think we support submessages for within responses, but if we do, I doubt we have used it in any current APIs.